### PR TITLE
Use local videos in tests

### PR DESCRIPTION
**Pull request by OmarShehata**
This is to alleviate some of the issues in https://github.com/AnalyticalGraphicsInc/cesium/issues/7249.

Loading the videos from external URIs is one of the reasons we know the build sometimes fails. Using a local copies makes these tests a lot faster and more reliable. 

The videos are scaled down versions of the original, so they're about ~20 kb each. 

@mramato want to merge this?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @OmarShehata!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/local-videos/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Specs/Core/VideoSynchronizerSpec.js [hunk 1]
```diff
@@ -24,17 +24,17 @@ defineSuite([
     function loadVideo() {
         var element = document.createElement('video');
         var source = document.createElement('source');
-        source.setAttribute('src', 'http://cesiumjs.org/videos/Sandcastle/big-buck-bunny_trailer.webm');
+        source.setAttribute('src', './Data/Videos/big-buck-bunny-trailer-small.webm');
```
Comment (mramato) [inline Specs/Core/VideoSynchronizerSpec.js]: Remove the `./`, it's unneeded (plus this is a url, not a filepath)

**Comment by mramato** [comment]
Thanks @OmarShehata, just that one comment and I'll merge.

**Comment by OmarShehata** [comment]
@mramato all good!

**Comment by mramato** [comment]
Thanks

**Pull request merged**